components/raftstore/src/store/metrics.rs

```diff
@@ -663,6 +663,12 @@ lazy_static! {
         exponential_buckets(8.0, 2.0, 24).unwrap()
     ).unwrap();
 
+    pub static ref CPU_REGION_VEC: HistogramVec = register_histogram_vec!(
+        "tikv_cpu_region",
+        "Histogram of CPU",
```

Comment:
Please make this description clearly. What is the relationship between cpu & region?